### PR TITLE
Galfit

### PR DESCRIPTION
**Pull request by SunilSimha**
GALFIT wrapper and additional helper functions to parse its output.

### CODE REVIEW

File: frb/galaxies/galfit.py [hunk 1]
```diff
@@ -0,0 +1,463 @@
...
+    Returns:
+    """
+    hdr = cutout.wcs.to_header()
+    imghdu = fits.PrimaryHDU(cutout.data, hdr)
+    hdulist = fits.HDUList([imghdu])
+    hdulist.writeto(filename, overwrite=overwrite)
+    return
+
+def _genconf(imgfile:str, psffile:str,
+            configfile:str=None, outdir:str=None, outfile:str=None,
```
Comment (profxj) [inline frb/galaxies/galfit.py:49]: whoa!  I've never seen this kind of arg list with defined variable types.

File: frb/galaxies/galfit.py [hunk 2]
```diff
@@ -0,0 +1,463 @@
...
+    return sky_dict
+
+def run(imgfile, psffile, platescale=0.125, **kwargs):
+    """
+    Run galfit. 
+    
+    Args:
+        imgfile (str): path to the image fits file.
+        psffile (str): path to the PSF model fits file.
+        platescale (float, optional): Size of pixel in arcsec.
```
Comment (profxj) [inline frb/galaxies/galfit.py:333]: I can't see a good reason to have a default.  
Better to have the user think about this explicitly.

File: frb/galaxies/galfit.py (hunk 1 above)
Comment (SunilSimha) [reply frb/galaxies/galfit.py:49]: Yeah, it's a neat documentation tool someone showed on the astrograds' Slack channel.

File: frb/galaxies/galfit.py (hunk 2 above)
Comment (SunilSimha) [reply frb/galaxies/galfit.py]: I did one better. It's no longer an input. The plate scale is read directly from the fits header.

**Comment by SunilSimha** [comment]
Galfit tests added and now the platescale is obtained directly from the image file without any user input.

File: frb/tests/test_galfit.py [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+from astropy.table import Table
+from astropy.wcs import WCS
+
+from pkg_resources import resource_filename
+
+from frb.galaxies.frbgalaxy import FRBHost
+
+try:
+    from frb.galaxies import galfit as glf
+except ImportError:
```
Comment (profxj) [inline frb/tests/test_galfit.py:19]: If this test is going to crash without this import, then best to have
it crash out here.

**Comment by profxj** [comment]
You also have to deal with the conflict noted on GitHub.
Do so by pulling in the `heintz2020` branch and resolve it.

File: frb/tests/test_galfit.py (hunk 3 above)
Comment (SunilSimha) [reply frb/tests/test_galfit.py]: Done.

**Comment by SunilSimha** [comment]
Merged.

**Pull request merged**